enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -218,6 +236,27 @@ An example IAM Trust Policy:
 
 ```
 
+#### External DNS Operator Details
+
+Prior to this enhancement, the [External DNS Operator API](https://github.com/openshift/external-dns-operator/blob/main/api)
+did not allow users to configure an AWS role ARN field. As a result, there wasn't a supported way to use the External
+DNS Operator to create DNS records in another AWS account within a shared VPC.
+
+The External DNS Operator configures and runs the [External DNS](https://github.com/openshift/external-dns) binary.
+Conveniently, External DNS already supports the [--aws-assume-role](https://github.com/openshift/external-dns/blob/fe00b4b83c2263282a9068655e8e3fbbc167b653/docs/faq.md#can-external-dns-manageaddremove-records-in-a-hosted-zone-which-is-setup-in-different-aws-account)
+argument, which uses the specified AWS role ARN when creating new DNS records. Therefore, to support shared VPCs with
+the External DNS Operator, we update the [API](#API-External-DNS-Operator) to expose the `--aws-assume-role` argument
+for the External DNS binary.
+
+By design, External DNS can update both public and private DNS zones. Therefore, External DNS Operator users are able
+to use both public and private DNS zones in other accounts, provided the role ARN has appropriate permissions.
```

Comment:
I don't think this statement logically follows without this addition:
```suggestion
By design, External DNS can update both public and private DNS zones using credentials that the cluster-admin provides. Therefore, External DNS Operator users are able
to use both public and private DNS zones in other accounts, provided the role ARN has appropriate permissions.
```

Reply:
Sure, provides clarity.